cmd/snap/inhibit.go

```diff
@@ -145,11 +143,24 @@ type inhibitionFlow interface {
 	FinishInhibitionNotification(ctx context.Context) error
 }
 
-var newInhibitionFlow = func(instanceName string) inhibitionFlow {
-	if isGraphicalSession() {
-		return &graphicalFlow{instanceName: instanceName}
+var newInhibitionFlow = func(cli *client.Client, instanceName string) inhibitionFlow {
+	if isGraphicalFlowSupported(cli) {
+		return &graphicalFlow{instanceName: instanceName, cli: cli}
```

Comment:
shouldn't we simply always emit the notice but do the fallback if there is no connection etc?

Reply:
Yes, good point